plugins/cicd-statistics/src/hooks/use-cicd-statistics-api.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useApi } from '@backstage/core-plugin-api';
+
+import { cicdStatisticsApiRef } from '../apis';
+
+export function useCicdStatisticsApi() {
```

Comment:
IIRC, I thought the default was a bit harsh, and I had an idea of a fake dataset to display instead, which never happened though. Perhaps I should spend a few minutes constructing such, so that people can see the UI although without real data, to understand what the plugin does. WDYT?

Reply:
As an aside, if you want to optionally get an an API instance, the canonical way is to do `useApiHolder().get(apiRef)`.